assets/src/components/routeLadder.tsx

```diff
@@ -14,6 +15,19 @@ interface Props {
   dispatch: Dispatch
 }
 
+const Header = ({ route, dispatch }: { route: Route; dispatch: Dispatch }) => (
```

Comment:
For my own learning: When would you use a component, created with `<Header ...args... />` and when would you use a function `{header(route, dispatch)}`?

Reply:
Lately I've been moving towards using more and more of the former because they feel more React-y and obviously like components. As far as I know though it is just a style thing. 🤷‍♀️